ci/fileUtil.sc

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env amm
 
+import $ivy.`javax.xml.bind:jaxb-api:2.3.1`
```

Comment:
I'm not sure it's as simple as that. Marathon builds and runs with Java 8. Maybe we should make sure that the correct JDK version is used.